src/assets/scripts/client/info/AirportInfoController.js

```diff
@@ -0,0 +1,287 @@
+
+        return this;
+    }
+
+    /**
+     * @for AirportInfoController
+     * @method reset
+     * @chainable
+     */
+    reset() {
+        this.wind = null;
+        this.altimeter = null;
+        this.elevation = null;
+        this.icao = null;
+
+        return this;
+    }
+
+    /**
+     * Updates the information taken from the AirportModel: the wind, the altimeter,
+     * and the elevation. Triggered on airport change.
+     *
+     * @for AirportInfoController
+     * @method onAirportChange
+     */
+    onAirportChange() {
+        const airport = AirportController.airport_get();
+        const icao = airport.icao.toUpperCase();
+        const windAngle = Math.round(radiansToDegrees(airport.wind.angle));
+
+        this.wind = this._buildWindAndGustReadout({ speed: airport.wind.speed, angle: windAngle });
+        this.altimeter = this._generateAltimeterReading(airport.wind.speed);
+        this.elevation = `${airport.elevation}`;
+        this.icao = icao;
+
+        this._render();
+    }
+
+    /**
+     * Sets the values from the updated airport info.
+     *
+     * @for AirportInfoController
+     * @method _render
+     * @private
+     */
+    _render() {
+        this.$windView.text(`${this.icao} ${this.wind}`);
+        this.$altimeterView.text(`${this.icao} ${this.altimeter}`);
+        this.$elevationView.text(`${this.icao} ${this.elevation}`);
+    }
+
+    /**
+     * Updates the clock, called from `AppController#update_pre`
+     *
+     * @for AirportInfoController
+     * @method updateClock
+     */
+    updateClock() {
+        this.simClockController.update();
+
+        const time = this.simClockController.render();
+
+        this.$clockView.text(time);
+    }
+
+    /**
+     * Formats the wind angle and speed from object into a string,
+     * in the format `${newAngle}${newSpeed}G${gustSpeed}`.
+     *
+     * @for AirportGameInfoView
+     * @method _buildWindAndGustReadout
+     * @param {Object} wind
+     * @returns {String} formatted string
+     * @private
+     */
+    _buildWindAndGustReadout(wind) {
+        const minGustStrength = 5;
+        const speed = wind.speed;
+        const angle = wind.angle;
+        const newAngle = leftPad((angle || 360), 3);
+        const newSpeed = leftPad(speed, 2);
+        // Creates a fake "gusting" speed
+        const gustStrength = speed * Math.random();
+        const gustSpeed = leftPad(Math.round(speed + gustStrength), 2);
+
+        if (gustStrength < minGustStrength) {
+            return `${newAngle} ${newSpeed}`;
+        }
+
+        return `${newAngle} ${newSpeed}G${gustSpeed}`;
+    }
+
+    /**
+     * Creates an 'altimeter' reading for the info view
+     *
+     * @for AirportInfoController
+     * @method _generateAltimeterReading
+     * @param {Number} windSpeed
+     * @returns {Number} the new altimeter value
+     * @private
+     */
+    _generateAltimeterReading(windSpeed) {
+        const DEFAULT_ALTIMETER_VALUE = 2992;
```

Comment:
this should be camelCase since it's local only to this method.  I feel like we have this value in constant somewhere already at a more global level, but maybe not

Reply:
moved to aircraftConstants